data/json/npcs/talk_tags_chat.json

```diff
@@ -111,18 +303,76 @@
     "text": [
       "<neutralchitchat>",
       "<thoughtfulchitchat>",
-      "How do these zombies even keep going?  What are they eating?  Do you think they'll ever rot away?",
-      "I been thinkin', one of these days, we're gonna run out of toilet paper.  What then?",
-      "Do you think it's weird how we'll, like, open a locked building and find a lone zombie inside?  How'd it even get there?",
-      "Sometimes I wonder if we're all psychos, not just the ones that went crazy and rioted.  I never would have thought I could do the things I've done since the world ended.",
-      "You read any good books lately?  I'm glad we still got books.",
-      "You know what I miss?  Movie theaters.  You think Hollywood survived this?  Maybe there's a bunch of zombie actors out there, filmin' shit out of reflex.  Hah, I'd watch that shit."
+      "How do these zombies even keep going? What are they eating? Do you think they'll ever rot away?",
+      "I been thinkin', one of these days, we're gonna run out of toilet paper. What then?",
+      "Do you think it's weird how we'll, like, open a locked building and find a lone zombie inside? How'd it even get there?",
+      "Sometimes I wonder if we're all psychos, not just the ones that went crazy and rioted. I never would have thought I could do the things I've done since the world ended.",
+      "You read any good books lately? I'm glad we still got books.",
+      "You know what I miss? Movie theaters. You think Hollywood survived this? Maybe there's a bunch of zombie actors out there, filmin' shit out of reflex. Hah, I'd watch that shit.",
+      "I wonder if anyone's tried to start a farm again. Gotta be tough with all the zombies, but food doesn't grow on trees.",
+      "Do you think there are still places untouched by all this? Like some remote island or hidden valley where people are just living normal lives?",
+      "Ever think about how quiet it is now? No cars, no planes, just the wind and the groans of the dead.",
+      "I miss music. Remember concerts? The energy of the crowd, the live sound. Wonder if any musicians survived.",
+      "It's strange how some buildings are completely untouched, like time just stopped in them. Then you go next door, and it's a total wreck.",
+      "You ever think about all the pets that were left behind? Poor things didn't ask for this any more than we did.",
+      "I used to complain about going to work. Now I'd give anything for a boring day at the office.",
+      "Do you think we'll ever get back to a point where we can have holidays again? Celebrating seems like a distant dream.",
```

Comment:
```suggestion
      "Sometimes I wonder if we're all psychos, not just the ones that went crazy and rioted.  I never would have thought I could do the things I've done since the world ended.",
      "You read any good books lately?  I'm glad we still got books.",
      "You know what I miss?  Movie theaters.  You think Hollywood survived this?  Maybe there's a bunch of zombie actors out there, filmin' shit out of reflex.  Hah, I'd watch that shit.",
      "I wonder if anyone's tried to start a farm again.  Gotta be tough with all the zombies, but food doesn't grow on trees.",
      "Do you think there are still places untouched by all this?  Like some remote island or hidden valley where people are just living normal lives?",
      "Ever think about how quiet it is now?  No cars, no planes, just the wind and the groans of the dead.",
      "I miss music.  Remember concerts?  The energy of the crowd, the live sound. Wonder if any musicians survived.",
      "It's strange how some buildings are completely untouched, like time just stopped in them.  Then you go next door, and it's a total wreck.",
      "You ever think about all the pets that were left behind?  Poor things didn't ask for this any more than we did.",
      "I used to complain about going to work.  Now I'd give anything for a boring day at the office.",
      "Do you think we'll ever get back to a point where we can have holidays again?  Celebrating seems like a distant dream.",
```